src/app/components/collection/collection-detail/collection-detail.component.ts

```diff
@@ -0,0 +1,345 @@
+import { CollectionReferenceDescription, CollectionReferenceUpdateOrg } from 'arlas-api';
+import { RoleData, UserOrgData } from 'arlas-iam-api';
+import {
+    ArlasCollaborativesearchService, ArlasIamService,
+    ArlasSettingsService, ArlasStartupService
+} from 'arlas-wui-toolkit';
+import { filter, finalize, mergeMap, of, switchMap } from 'rxjs';
+import { ConfirmModalComponent } from '../../confirm-modal/confirm-modal.component';
+import { MatIconRegistry } from '@angular/material/icon';
+import { DomSanitizer } from '@angular/platform-browser';
+
+@Component({
+    selector: 'arlas-collection-detail',
+    templateUrl: './collection-detail.component.html',
+    styleUrl: './collection-detail.component.scss'
+})
+export class CollectionDetailComponent implements OnInit {
+    @ViewChild('fieldTableSort', { static: true }) public sort: MatSort;
+
+    public collection: CollectionReferenceDescription;
+    public collectionName: string;
+    public fields: CollectionField[];
+
+    public collectionForm: FormGroup;
+
+    public displayedColumns = ['name', 'display_name', 'type', 'indexed', 'taggable'];
+    public isLoading = false;
+    public filterValue = '';
+
+    public isAuthentActivated: boolean;
+    public authentMode: 'openid' | 'iam';
+    public organisations: WritableSignal<UserOrgData[]> = signal([]);
+    public dataSourceFields: MatTableDataSource<any> = new MatTableDataSource([]);
+    public canEdit = false;
+    private connected = false;
+
+    public editMode = false;
+    public formInitialValues;
+
+    public constructor(
+        private readonly destroyRef: DestroyRef,
+        private readonly route: ActivatedRoute,
+        private readonly collabSearchService: ArlasCollaborativesearchService,
+        private readonly router: Router,
+        private readonly formBuilder: FormBuilder,
+        private readonly collectionService: CollectionService,
+        private readonly snackbar: MatSnackBar,
+        private readonly translate: TranslateService,
+        private readonly arlasIamService: ArlasIamService,
+        private readonly arlasStartupService: ArlasStartupService,
+        private readonly arlasSettingsService: ArlasSettingsService,
+        private readonly dialog: MatDialog,
+        private readonly iconRegistry: MatIconRegistry,
+        private readonly sanitizer: DomSanitizer
+    ) {
+        const authSettings = this.arlasSettingsService.getAuthentSettings();
+        this.isAuthentActivated = !!authSettings && authSettings.use_authent;
+        const isOpenID = this.isAuthentActivated && authSettings.auth_mode !== 'iam';
+        const isIam = this.isAuthentActivated && authSettings.auth_mode === 'iam';
+        if (isOpenID) {
+            this.authentMode = 'openid';
+        }
+        if (isIam) {
+            this.authentMode = 'iam';
+        }
+        iconRegistry.addSvgIcon('keyword', sanitizer.bypassSecurityTrustResourceUrl(location.origin + '/assets/keyword.svg'));
```

Comment:
has this icon any licence that we should mention ?

Reply:
No, I created it myself